.github/workflows/test.yml

```diff
@@ -40,14 +40,14 @@ jobs:
       - name: Run tests
         uses: Wandalen/wretry.action@v1.3.0
         with:
-          attempt_limit: 3
+          attempt_limit: 1
           action: reactivecircus/android-emulator-runner@v2
           with: |
             api-level: ${{ matrix.android_target }}
             target: ${{ matrix.emulator_type }}
             arch: x86_64
             force-avd-creation: true
-            emulator-options: -cores 2 -writable-system -no-snapshot-load -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
+            emulator-options: -cores 2 -writable-system -no-snapshot-load -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none -logcat-output seedvault_test_results/logcat.txt
```

Comment:
@stevesoltys do we keep this commit? Not sure if it worked and where to download the artifacts.

Reply:
It didn't seem to work, I don't think. It's probably fine to remove it for now.

Reply:
On a normal test run they will be under summary -

 https://github.com/seedvault-app/seedvault/actions/runs/7893712530?pr=621
(at the bottom)

Reply:
OK, I removed the logcat commit from this PR.